.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on:
+  push:
+    branches:
```

Comment:
I'm not quite sure we need to run this only on pushes to `master`?

I'd get rid of the

```yaml
branches:
  - master
```

part